lesson21_1/drink.js

```diff
@@ -0,0 +1,26 @@
+const drinksList = document.querySelector("#drinks-list");
+const liArray = Array.from(drinksList.children);
+
+function getUniqueElArray(elementsArray) {
+  let uniqueArray = [];
+
+  elementsArray.filter((text) => {
+    if (!uniqueArray.includes(text.textContent))
```

Comment:
That's nice. However, it'd be better to use `object` and its keys. Using `includes` inside `filter` equals using for inside for. We endeavor to avoid a loop inside a loop

Reply:
Add sort method